### PR TITLE
Implements the Grid/List toggle on profiles

### PR DESCRIPTION
**Pull request by colinta**
We need to get a TestFlight build of this for sure, to make sure Product gives it the OK.

### CODE REVIEW

File: Sources/Controllers/Profile/ProfileViewController.swift [hunk 1]
```diff
@@ -385,7 +389,19 @@ public class ProfileViewController: StreamableViewController {
         }
     }
 
-    private func userLoaded(user: User, responseConfig: ResponseConfig) {
+    func toggleGrid(isGridView: Bool) {
+        guard let user = user, responseConfig = streamViewController.responseConfig else { return }
+
+        let items = Array(streamViewController.dataSource.visibleCellItems[0..<3])
```
Comment (steam) [inline Sources/Controllers/Profile/ProfileViewController.swift]: Maybe we should name this "headerItems" or something so we remember what it is for in the future. Having been out of the project for a little bit one of the harder things to follow is our manipulation of the `StreamCellItem`'s. Just a thought.

File: Sources/Controllers/Profile/ProfileViewController.xib [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="9531" systemVersion="15C50" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="10117" systemVersion="15E65" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES">
```
Comment (steam) [inline Sources/Controllers/Profile/ProfileViewController.xib]: Oh man, I thought for sure that you'd get rid of this guy and `SnapKit` a `ProfileScreen` ❗

**Comment by steam** [comment]
A couple thoughts, otherwise LGTM.

File: Sources/Controllers/Profile/ProfileViewController.xib (hunk 2 above)
Comment (colinta) [inline Sources/Controllers/Profile/ProfileViewController.xib]: Yeah I didn't want to blow up this story that much.  A good idea, but this story was just a 1-pointer.

**Comment by steam** [comment]
Looks like a legit CI failure.

**Comment by steam** [comment]
I get this when I toggle between grid/list @colinta. Any ideas?

**Comment by colinta** [comment]
Yeah but is that what *product wants*!?

(haha - looking now)

File: Sources/Controllers/Profile/ProfileViewController.swift [hunk 3]
```diff
@@ -113,7 +111,13 @@ public class ProfileViewController: StreamableViewController {
         }
     }
 
-     deinit {
+    private func sharedInit() {
```
Comment (steam) [inline Sources/Controllers/Profile/ProfileViewController.swift]: 👍 ... was looking at this yesterday and wishing the shared initialization was consolidated.

**Comment by steam** [comment]
@colinta, I showed this to Lucian yesterday, did he talk with you about the padding between the bottom of the bio, the toggle and the space above the first post?

**Pull request merged**